src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OptionalAttribute, WritableAttribute, OptionalWritableAttribute, Attribute, Cluster, OptionalCommand, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvEnum, TlvField, TlvNullable, TlvObject, TlvUInt16, TlvUInt8 } from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.5.8 */
+export const OptionsBitmap = {
+    /** Command execution control - see 1.6.5.8.1. */
+    ExecuteIfOff: BitFlag(0), // TODO default: true.
+
+    /** level effect on color - see 1.6.5.8.2 */
+    CoupleColorTempToLevel: BitFlag(1),
+};
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.1 */
+const  MoveToLevelCommand = TlvObject({
+    level: TlvField(0, TlvUInt8.bound({min:0, max:254})),
+    transitionTime: TlvField(1, TlvNullable(TlvUInt16)),
+    optionsMask: TlvField(2, TlvBitmap(TlvUInt8, OptionsBitmap)),
+    optionsOverride: TlvField(3, TlvBitmap(TlvUInt8, OptionsBitmap)), // TODO: 0 Default - all optionsOverride
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.2 */
+const  MoveCommand = TlvObject({
+    moveMode: TlvField(0, TlvEnum<MoveMode>()),
+    rate: TlvField(1, TlvNullable(TlvUInt8)),
+    optionsMask: TlvField(2, TlvBitmap(TlvUInt8, OptionsBitmap)),
+    optionsOverride: TlvField(3, TlvBitmap(TlvUInt8, OptionsBitmap)),
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.3 */
+const  StepCommand = TlvObject({
+    stepMode: TlvField(0, TlvEnum<StepMode>()),
+    stepSize: TlvField(1, TlvUInt8),
+    transitionTime: TlvField(2, TlvNullable(TlvUInt16)),
+    optionsMask: TlvField(3, TlvBitmap(TlvUInt8, OptionsBitmap)),
+    optionsOverride: TlvField(4, TlvBitmap(TlvUInt8, OptionsBitmap)),
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.4 */
+const  StopCommand = TlvObject({
+    optionsMask: TlvField(0, TlvBitmap(TlvUInt8, OptionsBitmap)),
+    optionsOverride: TlvField(1, TlvBitmap(TlvUInt8, OptionsBitmap)),
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 1.6.6.5  */
+const MoveToClosestFrequencyCommand = TlvObject({
+    frequency: TlvField(0, TlvUInt16 ), // FIXME - Default 0, how to set TlvField default?
+})
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.4 */
+const features = {
+    /** Dependency with the On/Off cluster */
+    OnOff: BitFlag(0), // TODO default: true.
```

Comment:
I know that this is the name in the specs, but it is really terrible and significantly reduce code readability.
I suggest to use: onOffClusterDependency